resources/content/schema/admin/admin-auth.json

```diff
@@ -113,6 +118,14 @@
         "publish.transitioningMessage": "cr",
         "publish.transitioningProgress": "cr",
 
+        "project" : "crud",
```

Comment:
Admin auth extends user auth now.  If the user auth already has this, don't put it in the admin auth.